lsix

```diff
@@ -108,144 +107,149 @@ autodetect() {
 	If your terminal actually does support sixel, please file a bug
 	report at http://github.com/hackerb9/lsix/issues
 	EOF
-	read -s -t 1 -d "c" -p $'\e[c' >&2
-	if [[ "$REPLY" ]]; then
-	    echo
-	    cat -v <<< "Please mention device attribute codes: ${REPLY}c"
-	fi
+        read -s -t 1 -d "c" -p $'\e[c' >&2
+        if [[ "$REPLY" ]]; then
+            echo
+            cat -v <<<"Please mention device attribute codes: ${REPLY}c"
+        fi
 
-	exit 1
+        exit 1
     fi
 
     # SIXEL SCROLLING (~DECSDM) is now presumed to be enabled.
     # See https://github.com/hackerb9/lsix/issues/41 for details.
 
     # TERMINAL COLOR AUTODETECTION.
     # Find out how many color registers the terminal has
-    IFS=";"  read -a REPLY -s -t ${timeout} -d "S" -p $'\e[?1;1;0S' >&2
+    IFS=";" read -a REPLY -s -t ${timeout} -d "S" -p $'\e[?1;1;0S' >&2
```

Comment:
Can you please explain to me what you were thinking when you made this change?  Multiple spaces helps separate the environment from the command. Why change it?